### DISCUSSION TITLE
forget save, how can I get back all my 2 weeks work?

### DISCUSSION THREAD

**Discussion by fanweixiao** [excalidraw/excalidraw#3509] [Q&A]
Just found that I forget to save the work, I always keep the excalidraw tab and draw anytime. Is there a way I can get back way like from a temp file?

**Comment by dwelle** [comment]
Hey. Sadly, we don't store backups anywhere (that would effectively mean we'd have to version your diagrams) and I don't think you will be able to hunt for temp files either :(.

**Reply by fanweixiao** [reply]
Thanks for reply ;)
